synapse-grid-core/src/main/scala/ru/primetalk/synapse/core/dsl/Switcher2Dsl.scala

```diff
@@ -20,6 +20,7 @@ trait Switcher2Dsl extends SystemBuilderDsl{
         {
           case Signal(_, s@Signal(contact, _)) if outputContacts.contains(contact) =>
             Iterable.single(s)
+          case other => throw new IllegalStateException(s"Unexpected signal $other")
```

Comment:
In fact we can be more specific about error. We know that it's a `Signal` in data. And we can just say that `contact` of input signal is not in `outputContacts` list.

Reply:
done